wsclient/src/main/java/org/hpccsystems/ws/client/wrappers/gen/wssql/ArrayOfECLTimerWrapper.java

```diff
@@ -72,8 +72,9 @@ public String toString()
 			for ( int i = 0; i < this.local_eCLTimer.size(); i++)
 			{
```

Comment:
now that these values are null rather than initialized, won't loops like this potentially throw an NPE?

Reply:
line 69 prevents NPE, but yes let's keep an eye out for any potential reference to uninitialized vars.